source/Octopus.Client/Octopus.Client.nuspec

```diff
@@ -6,15 +6,15 @@
         <authors>Octopus Deploy</authors>
         <owners>Octopus Deploy</owners>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <licenseUrl>https://github.com/OctopusDeploy/OctopusClients/blob/master/LICENSE.txt</licenseUrl>
-        <projectUrl>http://octopusdeploy.com/</projectUrl>
-        <iconUrl>http://i.octopusdeploy.com/resources/Avatar3-32x32.png</iconUrl>
+        <license type="expression">Apache-2.0</license>
+        <projectUrl>https://github.com/OctopusDeploy/OctopusClients</projectUrl>
+        <icon>images\icon.png</icon>
```

Comment:
`iconUrl` is deprecated: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048